### PR TITLE
Update dars files

### PR DESCRIPTION
**Pull request by luisfponce**
* README.md: Remove no required folder creation
* Update baseline image to match DARS v0.1.2 version
* Update documentation to match DARS v0.1.2 version

### CODE REVIEW

File: dars/clearlinux/mkl/Dockerfile [hunk 1]
```diff
@@ -34,10 +37,10 @@ RUN URL='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15816' &
 # file exists on different layers. To minimize docker
 # image size, remove the overlapped files before copy.
 RUN mkdir /os_core_install
-COPY --from=clearlinux/os-core:latest / /os_core_install/
+COPY --from=base-image / /os_core_install/
```
Comment (DnPlas) [inline dars/clearlinux/mkl/Dockerfile]: Are you copying all root (/) from "base-image"? I yes, then you might not need all other COPYs in later layers. You can also avoid copying all root and copy tiny pieces instead.

File: dars/clearlinux/mkl/Dockerfile [hunk 2]
```diff
@@ -34,10 +35,10 @@ RUN URL='http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/15816' &
...
 RUN find / os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
 
-FROM clearlinux/os-core:latest
+FROM base-image
```
Comment (DnPlas) [inline dars/clearlinux/mkl/Dockerfile:29]: Did you ensure this FROM works?

Comment (luisfponce) [reply dars/clearlinux/mkl/Dockerfile:29]: Yes, before sent this PR I built up the images and perform sanity as CI/CD of DARS

File: dars/clearlinux/mkl/Dockerfile (hunk 1 above)
Comment (luisfponce) [reply dars/clearlinux/mkl/Dockerfile]: Make sense

**Comment by luisfponce** [comment]
can we merge this?

**Review by DnPlas — APPROVED** [review]

**Pull request merged**